### DISCUSSION TITLE
/passthrough-intel-igpu-with-gvt-g-to-a-vm-and-use-it-with-plex/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#172] [Announcements]
Passthrough Intel iGPU with GVT-g to a VM and use Quick Sync with Plex in docker on Proxmox
        
[https://blog.ktz.me/passthrough-intel-igpu-with-gvt-g-to-a-vm-and-use-it-with-plex/](https://blog.ktz.me/passthrough-intel-igpu-with-gvt-g-to-a-vm-and-use-it-with-plex/)

**Comment by smithj33** [comment]
Once you assigned a mediated device, say gvt-g_v5_4, to a VM, gvtg_v5_8 is no longer available to other VMs. It shows 0 of each available. If you stop the VM, then you can use a mediated device with another VM. Unfortunate.
